lib/src/services/test_data/issue_data.ts

```diff
@@ -11,8 +11,10 @@ export const issue: Issue = {
     project: {
       key: "project",
     },
+    // hierarchyLevel:0,
```

Comment:
What does this do?

Reply:
The REST API sends over a hirearchyLevel field, but it is not used in any of the codecs when converting it out of the JSON response. This seems to be unnecessary commented code